playground/docker-compose.fork.yml

```diff
@@ -0,0 +1,172 @@
+      dockerfile: ./docker/Dockerfile.binary
+    restart: on-failure
+    environment:
+      - FLYWAY_URL=jdbc:postgresql://db/?user=${POSTGRES_USER}&password=${POSTGRES_PASSWORD}
+    depends_on:
+      - db
+
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8082:8080
+    depends_on:
+      - db
+
+  orderbook:
+    build:
+      context: ../
+      target: orderbook
+      dockerfile: ./docker/Dockerfile.binary
+    restart: always
+    environment:
+      - NODE_URL=http://chain:8545
+      - DB_URL=postgres://db:5432/?user=${POSTGRES_USER}&password=${POSTGRES_PASSWORD}
+      - ENABLE_CUSTOM_INTERACTIONS=true
+      - ENABLE_PRESIGN_ORDERS=true
+      - ACCOUNT_BALANCES_SIMULATION=true
+      - ACCOUNT_BALANCES_SIMULATOR=Web3
+      - SIMULATION_NODE_URL=http://chain:8545
+      - EIP1271_SKIP_CREATION_VALIDATION=true
+      - ENABLE_EIP1271_ORDERS=true
+      - PRICE_ESTIMATORS=None
+      - NATIVE_PRICE_ESTIMATORS=baseline
+      - PRICE_ESTIMATION_DRIVERS=baseline|http://driver/baseline
+      - DRIVERS=baseline|http://driver/baseline
+      - ALLOW_PLACING_PARTIALLY_FILLABLE_LIMIT_ORDERS=true
+      - BIND_ADDRESS=0.0.0.0:80
+      - CHAIN_ID=$CHAIN
+      - BASELINE_SOURCES=None
+    depends_on:
+      - db-migrations
+    ports:
+      - 8080:80
+
+  autopilot:
+    build:
+      context: ../
+      target: autopilot
+      dockerfile: ./docker/Dockerfile.binary
+    restart: always
+    environment:
+      - DB_URL=postgres://db:5432/?user=${POSTGRES_USER}&password=${POSTGRES_PASSWORD}
+      - LOG_FILTER=warn,autopilot=trace,shared=info,shared::price_estimation=debug
+      - NODE_URL=http://chain:8545
+      - SIMULATION_NODE_URL=http://chain:8545
+      - SETTLE_INTERVAL=15s
+      - GAS_ESTIMATORS=Native,Web3
+      - PRICE_ESTIMATORS=None
+      - NATIVE_PRICE_ESTIMATORS=baseline
+      - BLOCK_STREAM_POLL_INTERVAL=1s
+      - ENABLE_LIMIT_ORDERS=true
+      - NATIVE_PRICE_CACHE_MAX_UPDATE_SIZE=100
+      - NATIVE_PRICE_CACHE_MAX_AGE=20m
+      - SOLVER_TIME_LIMIT=5
+      - PRICE_ESTIMATION_DRIVERS=baseline|http://driver/baseline
+      - DRIVERS=baseline|http://driver/baseline
+      - SKIP_EVENT_SYNC=true
+      - BASELINE_SOURCES=None
+      - ENABLE_COLOCATION=true
+    depends_on:
+      - orderbook
+
+  driver:
+    build:
+      context: ../
+      target: driver
+      dockerfile: ./docker/Dockerfile.binary
+    restart: always
+    command: --config /driver.toml
+    environment:
+      - ETHRPC=http://chain:8545
+      - ADDR=0.0.0.0:80
+    volumes:
+      - ./driver.toml:/driver.toml
+    ports:
+      - 9000:80
+
+  # baseline (solver engine)
+  baseline:
+    build:
+      context: ../
+      target: solvers
+      dockerfile: ./docker/Dockerfile.binary
+    restart: always
+    command: baseline --config /baseline.toml
+    environment:
+      - ADDR=0.0.0.0:80
+      - LOG=solvers=trace,shared=trace
+    volumes:
+      - ./baseline.toml:/baseline.toml
+    ports:
+      - 9001:80
```

Comment:
Could not bind port `9001` because `reth` uses that by default.

Reply:
As above